Cleaned the code, got rid of the french

tlangerak opened the pull request.

samuellapointe commented:
Hey, thanks for this. I created the project for a school class so I didn't intend to maintain it, only to keep it as is. I'll put a link to your project in the description

tlangerak commented:
Oops, was not to intend to send you the pull request! I am also using part of your code for school. Hope that is okay with you! (Notable changes are that we deleted the cube and added a spotify api)
